src/WeatherCard.js

```diff
@@ -49,6 +49,7 @@ const WeatherCard = ({ weather }) => {
   let region = "Taipei";
   let tempCity = "Linkou";
```

Comment:
It looks like you are trying to set default values. Replace these with a loading component